server/src/main/java/com/cloud/network/IpAddressManagerImpl.java

```diff
@@ -329,6 +329,9 @@ public class IpAddressManagerImpl extends ManagerBase implements IpAddressManage
 
     public static final ConfigKey<Integer> PUBLIC_IP_ADDRESS_QUARANTINE_DURATION = new ConfigKey<>("Network", Integer.class, "public.ip.address.quarantine.duration",
             "0", "The duration (in minutes) for the public IP address to be quarantined when it is disassociated.", true, ConfigKey.Scope.Domain);
+    
+    public static final ConfigKey<Boolean> AllowNestedVMAccess = new ConfigKey<>("Advanced", Boolean.class, "allow.nested.vm.access",
```

Comment:
@bradywilkin  
Append the new configKey "AllowNestedVMAccess"  to the list returned by the getConfigKeys() method in IpAddressManagerImpl. Please refer getConfigKeys().